src/api/version.ts

```diff
@@ -270,6 +271,57 @@ export namespace VersionHelper {
     return isCachedNewerVersionAvailable
   }
 
+  /**
+   * Returns true if verA > verB
+   */
+  export async function gtChectlVersion(verA: string, verB: string): Promise<boolean> {
+    return (await compareChectlVersions(verA, verB)) > 0
+  }
+
+  /**
+   * Retruns:
+   *  1 if verA > verB
+   *  0 if verA = verB
+   * -1 if verA < verB
+   */
+  async function compareChectlVersions(verA: string, verB: string): Promise<number> {
+    if (verA === verB) {
+      return 0
+    }
+
+    const verAChannel = verA.includes('next') ? 'next' : 'stable'
+    const verBChannel = verB.includes('next') ? 'next' : 'stable'
+    if (verAChannel !== verBChannel) {
+      // Consider next is always newer
+      return (verAChannel === 'next') ? 1 : -1
+    }
```

Comment:
What is the case where next and stable version are being compared?

Reply:
This is lib function that should handle all possible cases